### DISCUSSION TITLE
subscribeProvider is a dummy method in the latest AppKit

### DISCUSSION THREAD

**Discussion by timl16** [reown-com/appkit#3013] [Q&A]
**subscribeProvider** appears to be a dummy method in the latest AppKit. Previously, we used subscribeProvider to obtain information related to chain/account switching. It seems this functionality is no longer available. How can we restore this method? Additionally, the **allowUnsupportedChain** flag used to control the popup, but in the latest AppKit, it appears to always be turned on. Could someone please review these issues and provide a fix?

**Comment by timl16** [comment]
Latest AppKit implementation:

    subscribeProvider() {
        return null;
    }

**Comment by YOASOBIikura** [comment]
I had the same problem and now have to use the deprecated we3Modal API.
